content/docs/api-reference/get_url.md

```diff
@@ -30,7 +30,7 @@ specified by its `path` in a `repo` (<abbr>DVC project</abbr>), is stored.
 
 The URL is formed by reading the project's
 [remote configuration](/doc/command-reference/config#remote) and the
-[DVC-file](/doc/user-guide/dvc-file-format) where the given `path` is found
+[`.dvc` file](/doc/user-guide/dvc-file-format) where the given `path` is found
```

Comment:
This is an interesting one actually. The files checked by get/import can be either .dvc files or dvc.yaml so it's not clear which one should be used here to replace. Maybe we should use a general term in this case like "DVC file"... WDYT @shcheklein ?

Reply:
just specify both?

Reply:
Yeah, that's the easiest way probably:
```suggestion
[`dvc.yaml` or `.dvc` file](/doc/user-guide/dvc-file-format) where the given `path` is found
```
> p needs wrapping

^ @utkarshsingh99

Reply:
Actually, please separate each term and link both to the same page, the links will be updated later.

Reply:
> p needs wrapping
@jorgeorpinel I didn't get this. Could you explain?

Reply:
> p needs wrapping

This means that the suggestion I left on GH would break formatting rules. The paragraph needs wrapping (fit into 80 chars). You've already done this though, thanks. Resolving this one.